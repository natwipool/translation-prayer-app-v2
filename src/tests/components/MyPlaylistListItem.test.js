import React from 'react';
import { shallow } from 'enzyme';
import MyPlaylistListItem from '../../components/MyPlaylistListItem';
import transPrayers from '../fixtures/transPrayers';

test('should render MyPlaylistListItem with a playlist', () => {
  const wrapper = shallow(<MyPlaylistListItem {...transPrayers[0]}/>);
  expect(wrapper).toMatchSnapshot();
});