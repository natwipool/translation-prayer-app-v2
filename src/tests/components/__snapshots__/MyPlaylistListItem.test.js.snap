// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyPlaylistListItem with a playlist 1`] = `
<div>
  <h3
    className="list-item__title"
  >
    เขมาเขมสรณคมนปริทีปิกาคาถา
  </h3>
  <span
    className="list-item__sub-title"
  >
    บทสวดมนต์ภาคเย็น
  </span>
</div>
`;
